static/js/components/tiles/line_tile.tsx

```diff
@@ -144,7 +145,36 @@ export function LineTile(props: LineTilePropType): JSX.Element {
       replacementStrings={getReplacementStrings(props)}
       className={`${props.className} line-chart`}
       allowEmbed={true}
-      getDataCsv={chartData ? () => dataGroupsToCsv(chartData.dataGroup) : null}
+      getDataCsv={() => {
+        const denoms = props.statVarSpec.map((v) => (v.denom ? v.statVar : ""));
+        const entityProps = props.placeNameProp
+          ? [props.placeNameProp, ISO_CODE_ATTRIBUTE]
+          : undefined;
+        if ("enclosedPlaceType" in props && props.enclosedPlaceType) {
+          return datacommonsClient.getCsvSeries({
+            childType: props.enclosedPlaceType,
+            entityProps,
+            parentEntity: props.place.dcid,
+            perCapitaVariables: _.uniq(denoms),
+            startDate: props.startDate,
+            endDate: props.endDate,
+            variables: props.statVarSpec.map((v) => v.statVar),
+          });
+        } else {
+          const entities =
```

Comment:
hm, like with getting date from props, might also be a good idea to have a function for getting entities and use it in both the regular data fetch and the get csv since there's some logic involved.

Reply:
Done, and simplified the regular fetch data logic